lib/devcycle-ruby-server-sdk/localbucketing/bucketed_user_config.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+
+module DevCycle
+  class Environment
+    attr_accessor :id
+    attr_accessor :key
+  end
+
+  class FeatureVariationMap
+    attr_accessor :the_6216422850294_da359385_e8_b
+  end
+
+  class FeaturesTest
+    attr_accessor :id
+    attr_accessor :test_type
+    attr_accessor :key
+    attr_accessor :variation
+    attr_accessor :variation_name
+    attr_accessor :variation_key
+  end
+
+  class Features
+    attr_accessor :features_test
+  end
+
+  class EdgeDB
+    attr_accessor :enabled
+  end
+
+  class Colors
```

Comment:
why do we need to represent these things in the SDK types? ruby will never use this right

Reply:
It's part of the model for the bucketed user config, which may fail JSON deserialization if it's not present.

Reply:
we cant have it fail on unknown fields, it has to allow any additional field we add in the future

Reply:
May be able to remove; pending above too; Not sure if it's used.